db/chores/migrate_messages_to_scheduled_messages.rb

```diff
@@ -0,0 +1,91 @@
+class MigrateMessagesToScheduledMessages
+  def perform
+    execute("SELECT * FROM onboarding_messages").each do |row|
+      migrate_onboarding_message_to_scheduled_message(row)
+    end
+    execute("SELECT * FROM quarterly_messages").each do |row|
+      migrate_quarterly_message_to_scheduled_message(row)
+    end
+  end
+
+  private
+
+  def migrate_onboarding_message_to_scheduled_message(row)
+    # Migrate to new table
+    row["type"] = 0
+    values = ["created_at","updated_at","title","body","days_after_start","time_of_day","type","end_date","deleted_at"].map do |column|
+      sanitize(row[column])
+    end
+    scheduled_message_id = insert(
+      <<-SQL
+        INSERT INTO scheduled_messages (created_at, updated_at, title, body, days_after_start, time_of_day, type, end_date, deleted_at)
+        VALUES (#{values.join(",")})
+      SQL
+    )
+
+    # Update sent messages
```

Comment:
what about putting each of these into a well-named method rather than using comments? eg: lines 27 through 33 can be `def update_sent_messages` ?